app/views/tour_list/show.html.erb

```diff
@@ -0,0 +1,29 @@
+<h3>Tour List</h3>
+<% if tour_list.contents.empty? %>
+  <p>There are no landmarks currently in your tour list. Add some by visiting the landmark's show page and click on "Add to Tour List"</p>
+<% else %>
+
+  <ul id="landmarks">
+    <% tour_list.landmarks.each do |landmark, quantity| %>
+      <li id="landmark-<%= landmark.id %>" >
+        <%= link_to landmark.name, landmark_path(landmark) %>
+        <%= button_to "Remove Landmark", tour_list_landmark_path(landmark.id), method: :delete, class: 'btn btn-primary' %>
+        <% unless tour_recording.contents.value?(landmark.id) %>
```

Comment:
Change to if/else -- rearrange as needed

Reply:
I disagree, the way it is written now follows the workflow. If we change it to if/else, it will look confusing.